assets/tests/components/swingsView.test.tsx

```diff
@@ -9,7 +11,7 @@ import useVehiclesForRunIds from "../../src/hooks/useVehiclesForRunIds"
 import { Route, Swing } from "../../src/schedule"
 import { initialState, selectVehicle, toggleSwingsView } from "../../src/state"
 import { Vehicle, Ghost, VehicleOrGhost } from "../../src/realtime"
-import { HeadwaySpacing } from "../../src/models/vehicleStatus"
+// import { HeadwaySpacing } from "../../src/models/vehicleStatus"
```

Comment:
I think you can just delete this line now that we're getting the `HeadwaySpacing` from the factory?